### ISSUE TITLE
Crash in SE related to CME finishing on deleted surface.

### ISSUE THREAD

**Issue by EphDoering** [EarendelDevelopers/factorio-mods#264]
If a CME happens on a surface that was deleted then when it tries to print hat it's done, it crashes:

```
76    local surface = Zone.get_surface(tick_task.zone)
77    surface.print({"space-exploration.alert-cme-passed"})
```

```
The mod Space Exploration (0.6.94) caused a non-recoverable error.
Please report this error to the mod author.

Error while running event space-exploration::on_tick (ID 0)
__space-exploration__/scripts/solar-flare.lua:77: attempt to index local 'surface' (a nil value)
stack traceback:
	__space-exploration__/scripts/solar-flare.lua:77: in function 'tick_flare'
	__space-exploration__/control.lua:1412: in function 'callback'
	__space-exploration__/scripts/event.lua:15: in function <__space-exploration__/scripts/event.lua:13>
```

**Comment by EphDoering** [comment]
Viewing the surface to re-generate it before the CME passed prevented the crash.

**Comment by BurninSun** [comment]
Fixed in 0.6.102
